### ISSUE TITLE
'ufw allow' not a valid way to open access to Prometheus in 11.3 any more

### ISSUE THREAD

**Issue by nktl** [NetApp-StorageGRID/storagegrid-monitoring#3]
Readme suggests to run 'ufw allow 9090' command to open access to internal SG Prometheus instance, but that won't work in 11.3 any more.

The proper process is to ssh to the admin host as root and then:

1. Update prometheus bind IP to 0.0.0.0 (by default it listens on localhost - not accessible from outside):

- edit  /etc/sv/prometheus/run file, set --web.listen-address="0.0.0.0:9090" parameter
- restart Prometheus with 'sv restart prometheus'

2. Update nftables rule with following command:

`nft add element inet sgfilter external_ports { 9090 }`

Since this process is not mega user friendly and supportable, it would be great to get some official option in the SG Admin UI to enable external access to Prometheus.

**Comment by mschindl** [comment]
You are right. Please go in touch with your appropriate NetApp Account Team to get futher steps for solving it.
